src/helpers/graph-util.js

```diff
@@ -128,11 +128,16 @@ const GraphUtil = {
     graph.createLine("grid", minYAxisGrid, maxYAxisGrid, "black");
 
     if (showBoundingLabels) {
-      graph.createLabel("label", minXAxisGrid, minGridX.toString(), "left");
-      graph.createTick("tick", minXAxisGrid, "x");
-
-      graph.createLabel("label", maxXAxisGrid, maxGridX.toString(), "right");
-      graph.createTick("tick", maxXAxisGrid, "x");
+      
+      if(minGridY > -4) {
+        graph.createLabel('label', minXAxisGrid, minGridX.toString(), 'bottom left');
```

Comment:
I'm surprised flow allows `'bottom left' : AnchorT[]` does this work the same as `['bottom','left']`?

Reply:
since there's no local testing, I had to copy paste between node_modules and typed flow.  It is a copy/paste mistake.  I'm surprised flow didn't complain as well.  I will fix it to be safe.

Reply:
The lack of a test suite here is really unfortunate, I wonder how this was originally worked on? Maybe it was once in `frontend` proper and got moved out to try to get some OSS going?

Reply:
yes indeed, let's see if HS makes this a real OSS thing, graphy may come back to life.

Reply:
this is done